runtime/testing-basilisk/src/lib.rs

```diff
@@ -919,6 +946,9 @@ construct_runtime!(
 		Marketplace: pallet_marketplace = 109,
 		TransactionPause: pallet_transaction_pause = 110,
 
+		LiquidityMining: pallet_xyk_liquidity_mining = 111,
+		WarehouseLM: warehouse_liquidity_mining::<Instance1> = 112,
```

Comment:
```suggestion
		XYKWarehouseLM: warehouse_liquidity_mining::<XYKLiquidityMiningInstance> = 112,
```
would this be possible? (and would we want to specify the name of the instance on the left?)

Reply:
This didn't work for me(use instance name instead of `Instance1`) but maybe I did something wrong I'll try it again

Reply:
It's fine :shrug: